pkgs/development/python-modules/django-filter/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, fetchPypi
+, django
+, buildPythonPackage
+}:
+buildPythonPackage rec {
```

Comment:
Not sure whether you need `rec` here...

Reply:
Removing `rec` builds fine for me but the bots here keep complaining about output paths.